### PR TITLE
Add unique names for HttpNative and CryptoNative

### PR DESCRIPTION
**Pull request by eerhardt**
The methods exported from System.Net.Http.Native.* and System.Security.Cryptography.Native.* shims have short generic names. Making them more unique by adding "HttpNative_" and "CryptoNative_" prefix to each method name.

Partial fix for #4818.  Only Compression is left.
@stephentoub @sokket @bartonjs @nguerrera

### CODE REVIEW

File: src/Native/System.Security.Cryptography.Native/pal_ecdsa.cpp [hunk 1]
```diff
@@ -4,7 +4,8 @@
 #include "pal_ecdsa.h"
 #include "pal_utilities.h"
 
-extern "C" int32_t EcDsaSign(const uint8_t* dgst, int32_t dgstlen, uint8_t* sig, int32_t* siglen, EC_KEY* key)
+extern "C" int32_t
+CryptoNative_EcDsaSign(const uint8_t* dgst, int32_t dgstlen, uint8_t* sig, int32_t* siglen, EC_KEY* key)
```
Comment (bartonjs) [inline src/Native/System.Security.Cryptography.Native/pal_ecdsa.cpp]: Do we have style precedent for this formatting?

File: src/Native/System.Security.Cryptography.Native/pal_rsa.cpp [hunk 2]
```diff
@@ -145,23 +149,23 @@ static void SetRsaParameter(BIGNUM** rsaFieldAddress, uint8_t* buffer, int32_t b
...
+                                              uint8_t* p,
+                                              int32_t pLength,
+                                              uint8_t* dmp1,
+                                              int32_t dmp1Length,
+                                              uint8_t* q,
+                                              int32_t qLength,
+                                              uint8_t* dmq1,
+                                              int32_t dmq1Length,
+                                              uint8_t* iqmp,
+                                              int32_t iqmpLength)
```
Comment (bartonjs) [inline src/Native/System.Security.Cryptography.Native/pal_rsa.cpp]: (Or this indentation style)

**Comment by bartonjs** [comment]
Things look mechanical.  Assuming a tool did the formatting, LGTM.  (I noticed other whitespace changes, so I assume you just ran a format-everything script).

File: src/Native/System.Security.Cryptography.Native/pal_ecdsa.cpp (hunk 1 above)
Comment (eerhardt) [inline src/Native/System.Security.Cryptography.Native/pal_ecdsa.cpp]: The format-code.py script does this formatting.  I try not to fight it.

File: src/Native/System.Security.Cryptography.Native/pal_rsa.cpp (hunk 2 above)
Comment (eerhardt) [inline src/Native/System.Security.Cryptography.Native/pal_rsa.cpp]: again, the format-code.py script does this formatting.

**Comment by eerhardt** [comment]
In order to keep the CI running, I had to add back all of the original CryptoNative signatures.  These will be removed once the change is in, and the project.json files can be updated to a version with the new signatures.

**Comment by bartonjs** [comment]
Still LGTM with the double-declaration adapter.  If you keep it as a separate commit it'll be easy to just revert it later.

**Pull request merged**